docs/section10/10-7.md

```diff
@@ -17,120 +17,111 @@
+						- Check if capacity is greater or equal to 240000: `if total_cool_capacity_b >= 240000:`   
+							- Get baseline efficiency for the greater or equal to 240,000 Btuh size category:`COPnf_full_load_240Btuh_b = data_lookup(table_G3_5_2,240Btuh)`  
+							- Check if the efficiency is higher than the previous size category if so then set equal to the most conservative efficiency: `if COPnf_full_load_240Btuh_b > most_conservartive_eff: most_conservartive_eff = COPnf_full_load_240Btuh_b`  
                                                 
                 **Rule Assertion:**
-                - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or the modeled efficiency matches the most conservative efficiency across its size category and below and the efficiency was defined in the correct units then pass: `if efficiency_matches_acceptable_value == TRUE AND was_capacity_not_defined == FALSE: PASS`    
-                - Case 2: Elif capacity was not defined but the modeled efficiency meets the highest (most conservative) efficiency in the appropriate table based on system type then pass: `elif was_capacity_not_defined == TRUE AND capacity_not_defined_and_meets_most_conserv_eff_b ==  TRUE: PASS`  
-                - Case 3: Elif capacity was not defined and efficiency does not equal the highest (most conservative) efficiency in the appropriate table based on system type then UNDETERMINED: `elif was_capacity_not_defined == TRUE AND capacity_not_defined_and_meets_most_conserv_eff_b ==  FALSE: UNDETERMINED and raise_message "Check if the modeled baseline DX cooling efficiency was established correctly based upon equipment capacity and type."`          
-                - Case 4: Else, fail: `else: FAIL`  
+                - Case 1: if modeled efficiency equals the expected efficiency AND the capacity was defined AND the aggregation factor is either irrelevant or defined, then pass: `if modeled_efficiency_b == expected_baseline_eff_b AND not was_capacity_not_defined and zone_aggregation_factor_undefined_and_needed == FALSE: PASS`	
+                - Case 2: Elif capacity was not defined or and zone_aggregation_factor_undefined_and_needed equal TRUE but the modeled efficiency equals the highest (most conservative) efficiency in the appropriate table based on system type then UNDETERMINED: `elif (was_capacity_not_defined or zone_aggregation_factor_undefined_and_needed == TRUE) AND modeled_efficiency_b == most_conservartive_eff ==  TRUE: UNDETERMINED and raise message "Check if the modeled baseline DX cooling efficiency was established correctly based upon equipment capacity and type. The modeled efficiency matches the capacity bracket in Appendix G efficiency tables with the highest efficiency (i.e.,  most conservative efficiency has been modeled)."`   
```

Comment:
suggest indicating what was undefined - either capacity, zone_aggregation factor or both so that the user can make the required updates

Reply:
Depending on how this gets implemented, the RCT's reporting features may already be able to handle this feature of outputting the exact variables that were not defined without requiring the individual cases to be separated out. I will check with Weili on this as I am working on the implementation of this rule.

Reply:
sounds like a good approach